modules/account-lib/test/unit/coin/stx/transactionBuilder/transferBuilder.ts

```diff
@@ -202,6 +202,67 @@ describe('Stx Transfer Builder', () => {
       should.deepEqual(txJson.fee.toString(), '180');
     });
 
+    it('a transfer transaction signed multiple times with mid key no signer', async () => {
+      const builder = initTxBuilder();
+      builder.memo('test');
+      builder.sign({ key: testData.prv1 });
+     
+      builder.fromPubKey([testData.pub1, testData.pub2, testData.pub3]);
+      builder.numberSignatures(2);
+      const tx = await builder.build();
+      const txBuilder2 = factory.getTransferBuilder();
+      txBuilder2.from(tx.toBroadcastFormat());
+      txBuilder2.sign({ key: testData.prv3 });
+      //txBuilder2.fromPubKey([testData.pub1, testData.pub2, testData.pub3]);
+      const signedTx = await txBuilder2.build();
+      
+      const txJson = signedTx.toJson();
+      should.deepEqual(signedTx.toBroadcastFormat()
+      ,"808000000004012fe507c09dbb23c3b7e5d166c81fc4b87692510b000000000000000000000000000000b40000000302009473a37b914f703c81f33141d10eabe4550c4d61f113662cc11cdc0463fc377358408c73d3d3273cdbcc3511dbbd2031b5eaca4cb2b13925da9f9b0c7e64d1a600024abddd63b56c55cd1ed0803c26c473f5f0b9d8473b37b65bd812f035365f154b02016aac0347b8520d8905cbd13c601f45a0ccbbb24831320c54d9ee2c1e3656b76d75e1c527932267d80beb90257bbe2dc7184d9b168993ab3a40fa73be6973c5f5000203020000000000051a1ae3f911d8f1d46d7416bfbe4b593fd41eac19cb00000000000003e874657374000000000000000000000000000000000000000000000000000000000000"
```

Comment:
@wesgraham I can make a ref to Coins Sandbox where this tx was calculated. Should i?